example/README.md

```diff
@@ -0,0 +1,5 @@
+```
+spago bundle-app -m Example.Main -t example/example.js
+npm i -g parcel
+parcel serve example/index.html --open
+```
```

Comment:
Kinda terse, but better than nothing.

Reply:
Is parcel needed here? Can the file just be opened in the browser directly?

Reply:
It's not working for me without the bundler. This section is likely the issue:
https://github.com/purescript-contrib/purescript-ace/blob/23355ca6ca0ef9b85ca539330eaafdb6c1c49cfa/example/index.html#L19-L26
The path looks correct to me. Not sure if there's a non-bundler fix.

Reply:
Ok. If there’s a way to skip the bundled that would be ideal, but this is already a big improvement.